rastervision_pipeline/rastervision/pipeline/__init__.py

```diff
@@ -2,6 +2,25 @@
 import logging
 import json
 
+# This is in a try block since the pytorch_learner plugin (which installs these
+# dependencies) may not be installed.
+try:
+    # torch is imported before anything else in RV in order to avoid a
+    # segmentation fault when calling model.to('cuda'). This was determined empirically,
```

Comment:
Is it necessary for this to be in `pipeline` or can it also work if included before `import rastervision.pipeline` in `pytorch_learner`'s `__init__.py`?

Reply:
I tried putting it in `core` and `pytorch_learner` and it didn't fix the problem there.